diff -urN main.cpp main.cpp
--- /main.cpp	2017-09-26 02:26:26.000000000 +0900
+++ /main.cpp	2017-10-16 02:12:37.394449000 +0900
@@ -1289,11 +1289,11 @@
 
     // Check range
     if (bnTarget <= 0 || bnTarget > Params().ProofOfWorkLimit())
-        return error("CheckProofOfWork() : nBits below minimum work");
+        return false;//error("CheckProofOfWork() : nBits below minimum work");
 
     // Check proof of work matches claimed amount
     if (hash > bnTarget.getuint256())
-        return error("CheckProofOfWork() : hash doesn't match nBits");
+        return false;//error("CheckProofOfWork() : hash doesn't match nBits");
 
     return true;
 }
